app/org/sagebionetworks/bridge/dynamodb/DynamoStudyConsent1.java

```diff
@@ -18,20 +18,20 @@
 @DynamoDBTable(tableName = "StudyConsent1")
 public final class DynamoStudyConsent1 implements StudyConsent {
 
-    private String studyKey;
+    private String subpopGuid;
     private long createdOn;
     private boolean active;
     private String storagePath;
     private Long version;
 
     @Override
-    @DynamoDBHashKey
+    @DynamoDBHashKey(attributeName="studyKey")
     @JsonIgnore
```

Comment:
Hmmm. Maybe expose it because there's more than one in a study. Will remove annotation and update docs.